### PR TITLE
Update to react 16.0

### PR DESCRIPTION
**Pull request by diedsmiling**
Make library compatible with react 16.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/fraserxu/react-chartist",
   "peerDependencies": {
     "chartist": "^0.10.1",
-    "react": "^0.14.9 || ^15.3.0"
+    "react": "^0.14.9 || ^16.0.0"
```
Comment (fraserxu) [inline package.json]: This should still work with `15.*.*` too I think.

Comment (diedsmiling) [reply package.json]: Sure, fixed

**Comment by fraserxu** [comment]
Thanks @diedsmiling 👍

**Comment by diedsmiling** [comment]
Thanks for quick reaction. 👍

**Comment by fraserxu** [comment]
Published to v0.13.1

**Pull request merged**